Rakefile

```diff
@@ -36,3 +36,36 @@ if File.exist?(build_defs_file)
     end
   end
 end
+
+desc "verify that commit messages match CONTRIBUTING.md requirements"
+task(:commits) do
+  commits = ENV['TRAVIS_COMMIT_RANGE'] || 'master..HEAD'
```

Comment:
I am comfortable merging this as-is and fixing it up after some careful experimentation w/ Travis env. vars to try to minimize mangling.